src/migrateCalculatedMeasures.ts

```diff
@@ -18,20 +18,20 @@ export async function migrateCalculatedMeasures(
   legacyPivotFolder: ContentRecord
 ): Promise<ContentRecord> {
   const entitlements = legacyPivotFolder?.children?.entitlements;
-  const calculatedMeasuresFolder = entitlements?.children?.cm!;
+  const calculatedMeasuresFolder = entitlements?.children?.cm;
 
   const legacyCalculatedMeasures = calculatedMeasuresFolder
     ? await getCalculatedMeasures(calculatedMeasuresFolder)
     : [];
 
   const contentFolder = {
     entry: { isDirectory: true, owners: ["ROLE_USER"], readers: ["ROLE_USER"] },
-    children: {},
+    children: {} as { [id: string]: ContentRecord },
```

Comment:
Instead of casting here, can't you type `contentFolder` as `ContentRecord` ?
Same for `structureFolder`.